onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/Stop.java

```diff
@@ -26,18 +26,18 @@
 
   private static final long serialVersionUID = 1L;
   
-  private static final int MISSING_VALUE = -999;
+  public static final int MISSING_VALUE = -999;
 
   @CsvField(mapping = DefaultAgencyIdFieldMappingFactory.class)
   private AgencyAndId id;
 
   private String name;
 
-  @CsvField(mapping = LatLonFieldMappingFactory.class)
-  private double lat;
+  @CsvField(mapping = LatLonFieldMappingFactory.class, optional = true)
+  private double lat = MISSING_VALUE;
 
-  @CsvField(mapping = LatLonFieldMappingFactory.class)
-  private double lon;
+  @CsvField(mapping = LatLonFieldMappingFactory.class, optional = true)
+  private double lon = MISSING_VALUE;
```

Comment:
They're optional depending on the `location_type`: is there a way to reflect that in the class?

Reply:
There is no way to add this to field declation ([ref](https://github.com/OneBusAway/onebusaway-csv-entities/blob/master/src/main/java/org/onebusaway/csv_entities/schema/annotations/CsvField.java)), but I can throw exception if I detect such case during building the OTP graph. Do you think we should add this validation there?

Reply:
If there are some other similar validations already, it might be helpful. Otherwise, it is something that's covered by our existing GTFS validation.

Reply:
No, I don't think it has any validations like this as of now.